api/http/router.go

```diff
@@ -28,9 +28,10 @@ import (
 const (
 	APIURLStatus = "/status"
 
-	APIURLHealth     = "/api/v1/health"
-	APIURLWorkflow   = "/api/v1/workflow/:name"
-	APIURLWorkflowID = "/api/v1/workflow/:name/:id"
+	APIURLHealth        = "/api/v1/health"
+	APIURLWorkflow      = "/api/v1/workflow/:name"
+	APIURLWorkflowBatch = "/api/v1/workflow/:name/batch"
```

Comment:
@alfrunes It works, because `batch` is evaluated before the `:id`. We know that `:id` is a UUID we generate in the server, thus there cannot be a collision. Do you prefer to change the path to avoid any conflict? In this case, do you have a suggestion on the path?

Reply:
I think it works because the methods don't overlap, and we'll probably never have a GET endpoint for this path, so I'm ok with leaving it as is. My comment was just a premature warning to not repeat the mistake we did for the [deviceauth API](https://github.com/mendersoftware/deviceauth/blob/master/docs/management_api.yml#L372), but I digress.